package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "swagger2har",
-  "version": "0.1.2",
+  "version": "0.2.0",
```

Comment:
This is a minor bump

Reply:
Oh whoops you are right, I think it's best to go to 1.0